src/config/network/firewalls.md

```diff
@@ -0,0 +1,50 @@
+# Firewalls
+
+## iptables
+
+By default, the `iptables` package is installed on the base system. It provides
+[iptables(8)/ip6tables(8)](https://man.voidlinux.org/iptables.8).
+
+Two example rulesets are provided in the `/etc/iptables` directory:
+`empty.rules` and `simple_firewall.rules`.
+
+### Applying the rules at boot
+
+To apply iptables rules at runit stage 1, install the `runit-iptables` package.
+This adds a core-service which restores the rules in
+`/etc/iptables/iptables.rules` and `/etc/iptables/ip6tables.rules`.
+Alternatively, to apply these rules at stage 2, add the following to
+`/etc/rc.local`:
+
+```
+if [ -e /etc/iptables/iptables.rules ]; then
+  iptables-restore /etc/iptables/iptables.rules
+fi
+
+if [ -e /etc/iptables/ip6tables.rules ]; then
+  ip6tables-restore /etc/iptables/ip6tables.rules
+fi
+```
+
+After rebooting, check the active firewall rules:
+
+```
+# iptables -L
+# ip6tables -L
+```
+
+### Applying the rules at runtime
+
+`iptables` comes with a runit service to quickly flush or restore the rules on
+demand. Once the service is [enabled](../services/index.md#enabling-services),
+you can flush the rules with:
```

Comment:
Which rules does this use? If it's the ones mentioned in the section above, it might make sense to specify the rules used in the main `## iptables` part. Something like "The rules used in the solutions below are ..."

Reply:
Good point. i've just updated the PR; let me know what you think.

Reply:
LGTM